### PR TITLE
fix(components/pytorch): pytorch kfp components and Sample - GPU updates

### PR DESCRIPTION
**Pull request by shrinath-suresh**
This PR contains changes for adding compatibility with pytorch job operator. As of today, PyTorch Job operator works only with pytorch lightning master. Until the next release of PTL, installing PTL from source. 

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Comment by Bobgy** [comment]
nit: your title should be `fix(scope): title`. There is no space between fix and (scope). There is a `:` and a space after scope).

**Comment by shrinath-suresh** [comment]
> nit: your title should be `fix(scope): title`. There is no space between fix and (scope). There is a `:` and a space after scope).

Thanks Yuan. i will follow it from the next PRs.

File: samples/contrib/pytorch-samples/Dockerfile [hunk 1]
```diff
@@ -15,12 +15,16 @@ ARG BASE_IMAGE=pytorch/pytorch:latest
...
 
+RUN apt-get update && apt-get install -y git
+
 COPY . .
 
 RUN pip install --no-cache-dir -r requirements.txt
 
 RUN pip install --no-cache-dir pytorch-kfp-components
 
+RUN pip install --no-cache-dir git+https://github.com/PyTorchLightning/pytorch-lightning@master
```
Comment (chauhang) [inline samples/contrib/pytorch-samples/Dockerfile:26]: Are things working after this change for all the examples?

**Review by chauhang — APPROVED** [review]

**Pull request merged**